pkg/cosign/tuf/client.go

```diff
@@ -537,46 +563,32 @@ func (m *memoryCache) Set(p string, b []byte) error {
 	return nil
 }
 
-//go:embed repository
-var embeddedRootRepo embed.FS
-
-type embedded struct {
-	setImpl
-}
-
-func (e *embedded) Get(p string) ([]byte, error) {
-	b, err := embeddedRootRepo.ReadFile(path.Join("repository", "targets", p))
-	if err != nil {
-		return nil, err
+func (m *memoryCache) Get(p string) ([]byte, error) {
+	if m.targets == nil {
+		return nil, fmt.Errorf("no cached targets available, cannot retrieve %s", p)
 	}
-	// Unfortunately go:embed appears to somehow replace our line endings on windows, we need to switch them back.
-	// It should theoretically be safe to do this everywhere - but the files only seem to get mutated on Windows so
-	// let's only change them back there.
-	if runtime.GOOS == "windows" {
-		return bytes.ReplaceAll(b, []byte("\r\n"), []byte("\n")), nil
```

Comment:
Where did this go?

Reply:
Updated https://github.com/sigstore/cosign/pull/1921/commits/410d6cf9cef895df5fc7897abb0df8e48c00c803